### PR TITLE
Update quick-cd-tips.md

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
improve the readability of the insight

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/linux/user-and-file-management/shell/quick-cd-tips.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/linux/user-and-file-management/shell/quick-cd-tips.md?ref=5a455346d1f4523a6bebe6604d2d1a27518e7bff)

**Review by nem035 — APPROVED** [review]

File: linux/user-and-file-management/shell/quick-cd-tips.md [hunk 1]
```diff
@@ -90,12 +109,13 @@ To clear the stack, add the `-c` to `dirs`:
...
 ```
 
-Or, to undo the last change of directory and go back to last visited path:
+Or, to undo the last change of directory and go back to the last visited path:
 
 ```bash
 cd -
 ```
 
+To learn more about the directory stack or to view the differences between `popd` and `pushd` vs `cd` in both `bash` and `zsh`, check out the links in the "Learn More" section.
```
Comment (kapnobatai137) [inline linux/user-and-file-management/shell/quick-cd-tips.md:118]:
```suggestion
To learn more about the directory stack or to view the differences between `popd` and `pushd` vs `cd` in both `bash` and `zsh`, check out the links in the *Learn More* section.
```

File: linux/user-and-file-management/shell/quick-cd-tips.md [hunk 2]
```diff
@@ -24,59 +27,75 @@ links:
...
+
+If you add the -v flag you can include indexes and increase readability:
+```bash
 dirs -v
 0 ~
 1 ~/projects/project_1
 ```
 
-Note: the first item in the stack will always be the current directory.
+> The current directory is always the "top" of the directory stack.
```
Comment (kapnobatai137) [inline linux/user-and-file-management/shell/quick-cd-tips.md:66]:
```suggestion
> 💡 The current directory is always at *the top of the directory stack*.
```

File: linux/user-and-file-management/shell/quick-cd-tips.md [hunk 3]
```diff
@@ -24,59 +27,75 @@ links:
...
 ```bash
 pushd projects/
 ~/projects ~
 ```
 
-After calling `pushd`, the directory stack is logged in the shell.
+After calling `pushd` followed by a folder name, the directory stack is logged in the shell and you're switched to that folder.
 
-If you want to add the current directory to the stack, you would call `pushd`:
+> Tilde `~` is a short hand notation for `Users/your-name/`.
```
Comment (kapnobatai137) [inline linux/user-and-file-management/shell/quick-cd-tips.md:41]:
```suggestion
> 💡 Tilde `~` is a shorthand notation for `Users/your-name/`.
```

**Review by kapnobatai137 — CHANGES_REQUESTED** [review]
@Stefan-Stojanovic let's apply these suggestions and merge?

**Comment by enki-hq** [comment]
### Insight

| Insight Name | Error Message |
|:-:|:-:|
| quick-cd-tips | Insight has a practice question but it's missing the `practiceQuestion` metadata field. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight

| Insight Name | Error Message |
|:-:|:-:|
| quick-cd-tips | Insight has a practice question but it's missing the `practiceQuestion` metadata field. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight

| Insight Name | Error Message |
|:-:|:-:|
| quick-cd-tips | Insight has a practice question but it's missing the `practiceQuestion` metadata field. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Review by kapnobatai137 — APPROVED** [review]

**Pull request merged**